tests/src/pod-smoke.spec.ts

```diff
@@ -0,0 +1,177 @@
+      await pdRunner.screenshot('pods-run-image.png');
+      const containers = await runImage.startContainer(containerToRun);
+      await waitUntil(async () => containers.containerExists(containerToRun), 10000, 1000);
+      await pdRunner.screenshot('pods-container-exists.png');
+      const containerDetails = await containers.openContainersDetails(containerToRun);
+      await waitUntil(async () => {
+        return (await containerDetails.getState()) === ContainerState.Running;
+      }, 5000);
+    });
+
+    test(`Podify container '${containerToRun}'`, async () => {
+      const navigationBar = new NavigationBar(page);
+      const containers = await navigationBar.openContainers();
+      const createPodPage = await containers.openCreatePodPage(containerToRun);
+      await pdRunner.screenshot('pods-creation-page.png');
+      const pods = await createPodPage.createPod(podToRun);
+
+      await pods.heading.waitFor({ state: 'visible', timeout: 6000 });
+      await waitUntil(async () => await pods.podExists(podToRun), 10000, 1000);
+      const podDetails = await pods.openPodDetails(podToRun);
+      await waitUntil(
+        async () => {
+          return (await podDetails.getState()) === PodState.Degraded;
+        },
+        7000,
+        1000,
+      );
+      await pdRunner.screenshot('pods-pod-created.png');
+    });
+
+    test(`Checking pod details`, async () => {
+      const navigationBar = new NavigationBar(page);
+      const pods = await navigationBar.openPods();
+      await waitUntil(async () => await pods.podExists(podToRun), 5000, 500);
+      const podDetails = await pods.openPodDetails(podToRun);
+      await playExpect(podDetails.heading).toBeVisible();
+      await playExpect(podDetails.heading).toContainText(podToRun);
+      await pdRunner.screenshot('pods-pod-details.png');
+      await podDetails.activateTab('Logs');
+      await pdRunner.screenshot('pods-pod-details-logs.png');
+      await podDetails.activateTab('Summary');
+      const row = podDetails.getPage().getByRole('table').getByRole('row');
+      const nameText = await row.getByRole('cell').allInnerTexts();
+      playExpect(nameText).toContain(podToRun);
+      await pdRunner.screenshot('pods-pod-details-summary.png');
+      await podDetails.activateTab('Inspect');
+      await pdRunner.screenshot('pods-pod-details-inspect.png');
+      await podDetails.activateTab('Kube');
+      await pdRunner.screenshot('pods-pod-details-kube.png');
+    });
+
+    test(`Checking pod '${podToRun}' under containers`, async () => {
+      const navigationBar = new NavigationBar(page);
+      const containers = await navigationBar.openContainers();
+      expect(await containers.containerExists(`${podToRun} (pod)`)).toBeTruthy();
+      expect(await containers.containerExists(`${containerToRun}-podified`)).toBeTruthy();
+      await pdRunner.screenshot('pods-pod-containers-exist.png');
+    });
+
+    test.skip(`Stopping pod '${podToRun}'`, async () => {
```

Comment:
So let's make sure we use an image/container that can get into started mode? I would like to have a stopping feature of the pods covered.

Reply:
Changed testing container to nginx, which can be stopped manually.